kuma/javascript/src/payments/components/useScriptLoading.js

```diff
@@ -0,0 +1,40 @@
+// @flow
```

Comment:
file naming nit: this file is camelCase whereas we mostly use snake case all over the frontend files.

Reply:
I agree. But it's actually not "snake case" all over. It's "lower train case". Yeah, I made that up. E.g. `cancel-subscription-form.jsx` or `feedback-form.jsx`.